CONTRIBUTING.nd

```diff
@@ -0,0 +1,21 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make in the impact-makers-group Slack channel.
+
+Please check the [repo's issues page to see if your change has already been sugggested.](https://github.com/impactmakers/climatechoice/issues)
+
+Please note we have a code of conduct, please follow it in all your interactions with the project. It is linked [here.](https://docs.google.com/document/d/1pLXw1Qw69zeir3bTM3sbUkszy_RKN5qtWuP-jEbl5BU/edit?ts=5cd3049a#heading=h.2cu8fge9g0tm)
+
+## Pull Request Process
+
+<!---
+//TODO: come up with this
+-->
+#### Come up with this
+
+## Code of Conduct
+
+<!---
+//TODO: Link directly to CoC page or type out explicitly?
```

Comment:
I think a summary of the code of conduct / the main takeaways should be written here, with a link to the CoC link. But the link will suffice for now :)

Reply:
Can you move the code of conduct link here, or remove these comments?